Automattic/simplenote-macos
# Shortcuts: copy note content

### Fix
So one of the things that would be nice with shortcuts is to be able to get the content out of a note so you can apply it to another shortcut.  In this PR I have done that.  On it's own this shortcut isn't very useful, but if you connect it to another intent like copy to clipboard it becomes quite useful.

### Test
1. Run the branch and then open the shortcuts app.  Create a new shortcut and look for Simplenote in the apps
2. Add Copy Content From Note.  Select a note
3. search in the shortcuts sidebar for clipboard.  Add "Copy To Clipboard
4. run the shortcut.
5. Go to another text app and paste.  Confirm that the content from your note appears.

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
> These changes do not require release notes.
